backend/homework/2/src/main/java/org/example/StudentUtil.java

```diff
@@ -0,0 +1,56 @@
+package org.example;
+
+import java.lang.invoke.VarHandle;
+import java.time.temporal.ValueRange;
+
+public class StudentUtil {
+    public static double[] calculateGPA(int[] studentIdList, char[][] studentsGrades) {
+        double[] gpa=new double[studentIdList.length];
+        for(int i=0;i<studentsGrades.length;i++){
+            double sum=0;
+            for(int j=0;j<studentsGrades[i].length;j++){
```

Comment:
try using foreach loops for readability, use better names even in case of local scope vars